lib/active_event_store/subscriber_job.rb

```diff
@@ -41,7 +41,11 @@ def perform(payload)
       event = event_store.deserialize(**payload, serializer: ActiveEventStore.config.serializer)
 
       event_store.with_metadata(**event.metadata.to_h) do
-        subscriber.call(event)
+        if subscriber.is_a?(Class) && !subscriber.respond_to?(:call)
```

Comment:
We should probably add a note to the documentation, explaining this behaviour.

It could be confusing in case a subscriber class has both `.call` and `#call`.

Reply:
Agreed. Addressed in [57237c1](https://github.com/palkan/active_event_store/pull/9/commits/57237c123e85dc32c6cabbfd1375fd39c591c233). Let me know if the wording is good. :+1: